examples/all-clusters-minimal-app/all-clusters-common/all-clusters-minimal-app.zap

```diff
@@ -14302,6 +14428,92 @@
               "maxInterval": 65344,
               "reportableChange": 0
             }
+          ],
+          "events": [
+            {
+              "name": "SupplyVoltageLow",
```

Comment:
All events in this cluster are optional, no?

Reply:
The XML file doesn't include optional field for all events in the Pump Configuration and Control cluster, which then by default interpreted as mandatory and not optional events:
`./src/app/zap-templates/zcl/data-model/chip/pump-configuration-and-control-cluster.xml`

Reply:
The XML is wrong.

Reply:
Ticket to followup: #24884